frontend/public/components/modals/create-namespace-modal.jsx

```diff
@@ -114,6 +116,23 @@ const CreateNamespaceModalWithTranslation = connect(
         [allow]: t('public~No restrictions'),
         [deny]: t('public~Deny all inbound traffic'),
       };
+
+      const popOverText = () => {
```

Comment:
Naming nit: popover is one word, so `popoverText`

Reply:
Changed